tests/test_slogpy_file.py

```diff
@@ -0,0 +1,21 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog # noqa: N813 I001
+import tempfile
+
+"""This is a setup function"""
+def setup_function():
+    """create a file for output"""
+    slog.initialize(file_logging=True)
+
+"""debug method unit test"""
+def test_debug_file(msg="This is a file test"):
+    """The test for the debug function."""
```

Comment:
Since debug is special, it might make sense to test it specifically.  Since this test is about logging to file, perhaps it would make sense to start with info.

Reply:
I'm not sure if I understood this comment correctly because Alex was delegated to do a unit test for slog.info. What did you mean "start with info" and "test it specifically". I need more clarification. Thank you!